### DISCUSSION TITLE
key combinations

### DISCUSSION THREAD

**Discussion by sudden-snowball** [0x7c13/Notepads#1326] [Q&A] [answered]
Good day! Please tell me if there are keyboard shortcuts in the program?) For example, is it possible to use a keyboard shortcut to enable/disable Text Wrapping (Wrap word)?

**Comment by 0x7c13** [comment, marked as answer]
![image](https://github.com/0x7c13/Notepads/assets/1166162/79555ce9-080b-4842-91e3-1ddf23c14985)

**Reply by sudden-snowball** [reply]
thank you. I didn't notice that 😅

**Reply by minus21grams** [reply]
Any other keyboard shortcuts we're unaware of?
Mainly Lowercase, Uppercase and Capitalize conversions.
